File: Store.API/Store.DOMAIN/Produto/Model/ProdutoDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Store.DOMAIN.Model
{
    public class ProdutoDto
    {
        public long IdProduto { get; set; }
        public string DscProduto { get; set; }
        public double VlrUnitario { get; set; }
        public int QtdVenda { get; set; }
    }
}

File: Store.UI/Store/src/app/domain/produto/produtoDto.ts
import { Venda } from "../venda/venda";
import { Produto } from "./produto";

export class ProdutoDto {
  IdProduto : Number = 0;
  DscProduto : string = "";
  VlrUnitario : Number = 0;
  QtdVenda : Number = 1;
}
